### ISSUE TITLE
Zeiss CZI interleaved RGB images do not read as expected

### ISSUE THREAD

**Issue by NHPatterson** [PolusAI/bfio#10]
I was playing around with this library and it's really nice! I tried a few different images with success and found that RGB interleaved images from a Zeiss slide scanner had issues.

data in question:
https://drive.google.com/file/d/1DhnEFMOMvb4Zt5aqUe_2nfE9PrnU5uNE/view?usp=sharing
### example
```python
from bfio import BioReader
import napari

data_filepath = "testdata/czi_rgb.czi"
br = BioReader(data_filepath)

tout = br[:]
# tout.shape -> (7662, 15778, 1, 3, 1)

viewer = napari.Viewer()
viewer.add_image(np.squeeze(tout))
```
### expected output (QuPath):
![image](https://user-images.githubusercontent.com/17855764/132583912-7299ba57-80ed-460a-9c33-12d2a2cea422.png)

### BR output in napari:
![image](https://user-images.githubusercontent.com/17855764/132584197-9d41f0a7-37de-40a9-a834-f99409b94e23.png)
zoom:
![image](https://user-images.githubusercontent.com/17855764/132582889-17a15aa5-60cb-4996-a928-decb3be94997.png)

Thought I'd bring this to your attention.. I tested 2 RGB images from other vendors (Huron(bigTIFF), .scn(Leica)) and the RGB data came out correctly!

**Comment by Nicholas-Schaub** [comment]
I think we've run into this problem before, and it has to do with not handling interleaved RGB data properly. This is definitely a bug and should be resolved soon.

Thank you for sharing test data and code for reproducing the problem.

**Comment by Nicholas-Schaub** [comment]
@NHPatterson, I just fixed the problem and tested against your czi file. Could you send links to your other two test files so I can make sure my fix didn't undo what was already working?

**Comment by Nicholas-Schaub** [comment]
I was able to download a test scn file, and it looks like my change made the scn file load incorrectly. What it looks like is happening is that either czi or Bioformats is reversing the byte order of interleaved channels. I think I have to create a special catch just for czi files, which seems extremely odd to me.

**Comment by Nicholas-Schaub** [comment]
Resolved by #29
